x-pack/plugins/enterprise_search/public/applications/app_search/utils/formatted_date_time/index.tsx

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { FormattedDate, FormattedTime } from '@kbn/i18n/react';
+
+interface Props {
+  date: Date;
+  hasTime?: boolean;
```

Comment:
I think this should be maybe named `includeTime`? "hasTime" to me would be like, a function I run on a class to return a boolean depending on whether it has "time".

Reply:
yeah for sure, what do you think about `shouldShowTime`? or maybe just `showTime`, but I was worried that maybe would be confusing for like, the concept of a showtime lol.

Or maybe `hideTime` which defaults to false?

Reply:
I went with `hideTime` for now, we can definitely revisit later / feel free to open a new PR for changes ✨